spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkDataFile.java

```diff
@@ -191,6 +192,11 @@ public DataFile copy() {
     throw new UnsupportedOperationException("Not implemented: copy");
   }
 
+  @Override
+  public DataFile copyWithSpecificStats(Collection<Integer> statsToKeep) {
```

Comment:
I don't think this is needed when there's a default impl in the interface

Reply:
Yeah.. removed.
Thx